mkdocs.yml

```
@@ -0,0 +1,92 @@
+site_name: Med-ImageTools Documentation
+site_url: https://bhklab.github.io/med-imagetools
+
+
+markdown_extensions:
+  - pymdownx.highlight:
+      anchor_linenums: true
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
+  - pymdownx.superfences
+  
+theme: 
```

yamllint

[error] trailing spaces

(trailing-spaces)